packages/@react-spectrum/table/test/Table.test.js

```diff
@@ -3866,7 +3883,7 @@ describe('TableView', function () {
       fireEvent.scroll(scrollView);
       act(() => {jest.runAllTimers();});
 
-      expect(onLoadMore).toHaveBeenCalledTimes(3);
+      expect(onLoadMore).toHaveBeenCalledTimes(1);
```

Comment:
I could see how the first one is not scrolling far enough to cause a `onLoadMore` to call, but shouldn't the second one? I'd want to see `expect(onLoadMore).toHaveBeenCalledTimes(0);` after the first two scrolls. Should there be a test after this that does call `onLoadMore` twice?